python/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+cd /home/dev || exit
+mkdir -p build
+pipenv run bash -c "cd build && cmake .. && make DESTDIR=/home/dev install"
```

Comment:
As a note, since this is my first time using it: `DESTDIR` is used here to augment the default installation prefix.

Normally installation happens to the following path: CMAKE_INSTALL_PREFIX + DESTDIR + INSTALL_PATH

The CMAKE_INSTALL_PREFIX is `/usr/local`, DESTDIR is null and the INSTALL_PATH is given by the cmake install command (in this case `lib`). But, I don't want to install to `/usr/local/lib` because a) it requires root permission and b) it's less obvious to extract. Instead, I set the CMAKE_INSTALL_PREFIX to `/` in the cmakelist and prepend the destination dir with `/home/dev` here. It's not a super neat solution but it does the job, and it's nice to know as an option.